src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -156,7 +156,7 @@ IntPtr overlapped
             );
 
             // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
FWIW, at least one user was using it in a windows 8 store app (no idea why), but a bug was reported via nugget feedback and subsequently fixed. I recall that ReadFile via kernel32.dll was actually allowed on win8 when I checked with @ericstj. Did that change with UWP?